pkg/operator/ceph/csi/spec.go

```diff
@@ -428,3 +433,54 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string) error {
 
 	return err
 }
+
+// ValidateCSIVersion checks if the configured ceph-csi image is supported
+func ValidateCSIVersion(clientset kubernetes.Interface, namespace, rookImage, serviceAccountName string) error {
+	var ownerRef metav1.OwnerReference
+	timeout := 15 * time.Minute
+
+	logger.Infof("detecting the ceph csi image version for image %s", CSIParam.CSIPluginImage)
+
+	pod, err := k8sutil.GetRunningPod(clientset)
+	if err != nil {
+		return errors.New("could not get the rook operator pod to obtain the owner reference")
+	}
+	if pod == nil || len(pod.GetOwnerReferences()) == 0 {
+		return errors.Wrapf(err, "empty owner reference in rook operator pod")
+	}
```

Comment:
the error need to be interchanged  
```suggestion
pod, err := k8sutil.GetRunningPod(clientset)
	if err != nil {
return errors.Wrapf(err, "empty owner reference in rook operator pod")
		
	} 
if pod == nil || len(pod.GetOwnerReferences()) == 0 {
		return errors.New("could not get the rook operator pod to obtain the owner reference")
}

Reply:
@shaas can you please correct as suggested  above

Reply:
@Madhu-1 Oh sorry, yes. Changed that.